ros-tutorials/lesson-0-introduction.md

```diff
@@ -1,24 +1,75 @@
 
-To run ROS, we will be using Docker. This is a software that allows you to run programs inside an isolated environment. This means that you do not need to download anything but the docker client, and the particular image we will be using. First, download the appropriate docker client:
+## Docker Setup
+
+To run ROS, we will be using Docker. This is a software that allows you to run programs inside an isolated environment called a container. So, you only need to download the Docker client and then run our image. Our image includes all of the little pieces of software we need, rather than installing them individually. If you're curious, learn more [here](https://www.docker.com/resources/what-container) or [here](https://docs.docker.com/get-started/).
+
+Download and install the appropriate Docker client:
 
 * [Mac](https://docs.docker.com/docker-for-mac/install/)
-* [Windows (Home)](https://docs.docker.com/toolbox/toolbox_install_windows/)
-* [Windows (Other)](https://docs.docker.com/docker-for-windows/install/)
-* Which Windows do I have? Start > type "PC Info" > note if you have Windows 10 Home, Pro, Education, etc.
+* [Windows (Home)](https://docs.docker.com/docker-for-windows/install-windows-home/)
+* [Windows (Pro, Education, etc.)](https://docs.docker.com/docker-for-windows/install/)
+
+:information_source: Which Windows do I have? *Settings > System > About > look at "Edition"*<br>
+:warning: If you have Windows Home, make sure you follow the installation instructions exactly. Instead, we recommend upgrading to Windows Education using [Duke's free student license](https://software.duke.edu), though it is not required.
+
+Once the Docker client is installed, run it like you would any other app on your computer.
+
+## Starter Code and Git
+
+To share and manage versions of our source code, we use [Git](https://git-scm.com/). When you have a folder of code files for your project, Git calls it a repository. We're going to use Git now to get some starter code for these tutorials.
+
+Open up Terminal in Mac or Command Prompt in Windows. Type `git --version` and press Enter. If the command is not found, close the window and install Git from [here](https://git-scm.com/downloads). Then re-open Terminal or Command Prompt.
+
+Now, run the following command:
+
+```bash
+git clone https://github.com/DukeRobotics/documentation.git
+```
+
+This clones, or makes a copy of, our documentation repository (the one you're reading this tutorial in right now). It contains the starter code.
+
+## Run Docker
+
+Now, we will download and run our Docker image so we can use that isolated environment.
+
+Run the following to change directory to the starter code:
+```bash
+cd documentation/ros-tutorials  # Mac
+cd documentation\ros-tutorials  # Windows
```

Comment:
90% sure that `cd documentation/ros-tutorials` also works for Windows, so you unify these.

Reply:
You're right, just tested again. Not sure what I was doing differently last time I tested.